### ISSUE TITLE
Multiple aggregate columns not showing on Pinot Data Explorer.

### ISSUE THREAD

**Issue by keenangebze** [apache/incubator-pinot#4542]
I just built quickstart from latest commit from origin/HEAD: 83cc88d76298edafbd0ca72a7d2fbde8e13dabf0

Do some queries on baseballStats table:
```
select teamID, COUNT(*), MAX(numberOfGames) from baseballStats group by teamID
``` 

But the table view doesn't show the max number of games column (image 1). But, the JSON view returns (image 2).

Images are attached:
**Image 1:**
![Screenshot from 2019-08-20 21-13-56](https://user-images.githubusercontent.com/49970715/63355215-ff270e00-c38f-11e9-8f8b-e22fe6677223.png)

**Image 2:**
![Screenshot from 2019-08-20 21-14-05](https://user-images.githubusercontent.com/49970715/63355162-e7e82080-c38f-11e9-83b8-7a33b2168089.png)

**Comment by kishoreg** [comment]
Yes. The table in the query console is buggy and needs some work. Feel free to contribute.

**Comment by keenangebze** [comment]
I see. Thanks for the quick reply.
